check_prerequisites.sh

```diff
@@ -164,7 +164,7 @@ if ! [[ "$STATUS" == "\"HEALTHY\"" ]]
 then
     echo -e "${RED}It seems that CAS '$CAS' in namespace '$CAS_NAMESPACE' is not healthy: status is $STATUS${NC}"
     echo -e "- ${ORANGE}- You can install cas as follows: kubectl provision cas $CAS $CAS_NAMESPACE -v${NC}"
-    error_exit
+    # error_exit
```

Comment:
Don't just comment out, if it should be removed. But why should it not error_exit here?

Reply:
Done